stable/redis-cluster/README.md

```diff
@@ -0,0 +1,76 @@
+# Redis
+
+[Redis](http://redis.io/) is an advanced key-value cache and store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets, sorted sets, bitmaps and hyperloglogs.
+
+## TL;DR;
+
+```bash
+$ helm install stable/redis-cluster
+```
```

Comment:
Can you provide a little detail on what this chart installs in terms of the redis roles? eg. sentinel, master, node